spec/site_prism/page_objects/pages/admin/cases/foi_page.rb

```diff
@@ -0,0 +1,41 @@
+module PageObjects
+  module Pages
+    module Admin
+      module Cases
+        class FOIPage < PageObjects::Pages::Base
```

Comment:
Should this be `NewFOIPage` so that it follows the path (`/admin/cases/new/foi`) and also matches what we have for non-admin FOI creation?